extensions/openapi-generator/deployment/src/main/resources/templates/pojo.qute

```diff
@@ -0,0 +1,145 @@
+{#if withXml}
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlType;
+import javax.xml.bind.annotation.XmlEnum;
+import javax.xml.bind.annotation.XmlEnumValue;
+{#else}
+import java.lang.reflect.Type;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+{/if}
+
+{#if withXml}
+@XmlAccessorType(XmlAccessType.FIELD)
+{#if m.hasVars}@XmlType(name = "{m.classname}", propOrder =
+    { {#for var in m.vars}"{var.name}"{#if hasNext}, {/if}{/for}
+}){#else}
+@XmlType(name = "{m.classname}")
+{/if}
+{#if !m.parent || m.parent.isEmpty}@XmlRootElement(name = "{m.classname}"){/if}
+{#else}
+@JsonIgnoreProperties(ignoreUnknown = true)
+{/if}
+{#if m.description}
+/**
+  * {m.description}
+ **/
+{/if}
+{#include additionalModelTypeAnnotations.qute m=m/}
+public class {m.classname} {#if m.parent}extends {m.parent}{/if}{#if m.serializableModel} implements Serializable{/if} {
```

Comment:
This looks like we're generating getter/setters for the POJOs, but these days we tend to write POJOs using public fields, perhaps we need to change that?

Reply:
Yeah, these are accessors. I can get rid of them and change to public fields. np.